block.css

```diff
@@ -0,0 +1,58 @@
+.block1{
+    background-color: #2A9D8F;
+    display: block;
+    width: 10vh;
+    height: 15vh;
+    border-radius: 10%;
+    margin:2% 0
+    
+}
+.block1:hover{
+    transition-property: all;
+    transform: scale(4,1);
+    
+
+}
+.block2{
+    background-color:#e9c46a;
+    display: block;
+    width: 10vh;
+    height: 15vh;
+    border-radius: 10%;
+    margin:2% 0
```

Comment:
Things can improve: 
1. block2 has many similar CSS properties compared with block1.  e.g. display, width, height, border-radium and margin. So you can extract these same properties in one class. And for each block, you can just define there background-color.